lib/generators/react_on_rails/setup_files_generator.rb

```diff
@@ -0,0 +1,26 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class SetupFilesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def update_git_ignore
+        append_to_file ".gitignore" do
+          "# React on Rails\nnpm-debug.log\nnode_modules"
+        end
+      end
+
+      def update_application_js
+        insert_into_file "app/assets/javascripts/application.js",
+                         before: "//= require_tree ." do
```

Comment:
There's no guarantee that `require_tree .` will be in `application.js`. I've never used that.

This is what we have in the tutorial:

```
// Need to be on top to allow Poltergeist test to work with React.
//= require es5-shim/es5-shim
//= require react_on_rails

// It is important that generated/vendor-bundle must be before bootstrap since it is exposing jQuery and jQuery-ujs
//= require generated/vendor-bundle
//= require generated/app-bundle

//= require bootstrap-sprockets
//= require turbolinks
```